server/game/stateWatchers/CardsLeftPlayThisPhaseWatcher.ts

```diff
@@ -0,0 +1,42 @@
+import { Card } from '../core/card/Card';
+import { UnitCard } from '../core/card/CardTypes';
+import { UpgradeCard } from '../core/card/UpgradeCard';
+import { StateWatcherName } from '../core/Constants';
+import Player from '../core/Player';
+import { StateWatcher } from '../core/stateWatcher/StateWatcher';
+import { StateWatcherRegistrar } from '../core/stateWatcher/StateWatcherRegistrar';
+
+export interface CardLeftPlayEntry {
+    card: UnitCard | UpgradeCard;
+    controlledBy: Player;
+}
+
+export type ICardsLeftPlayThisPhase = CardLeftPlayEntry[]
```

Comment:
nit: missing semicolon, iirc there's a linter bug that prevents us from just adding that as a thing it catches automatically